### PR TITLE
Update step-1.mdx

### PR DESCRIPTION
**Pull request by plachy-jozef**
Closes #

Hi,
it took me quiet some time to run the step-1 part of the tutorial, the section about icons namely.
After digging into documentation of `carbon-component-angular` module I was able to make the icons working properly.
Therefore you would like to propose couple changes in this part of the step-1 documentation.

#### Changelog

**Changed**

- couple paragraphs in `Import and register icons` section

### CODE REVIEW

File: src/pages/developing/angular-tutorial/step-1.mdx [hunk 1]
```diff
@@ -301,34 +301,46 @@ package.
...
+
+imports: [
+  ...
+  IconModule
+]
+```
+<!-- prettier-ignore-end -->
 
-Then we need to add the template code. Populate `header.component.html` with:
+Now the icon is ready to be used in template code. Template in `header.component.html` should look like this:
```
Comment (Akshat55) [inline src/pages/developing/angular-tutorial/step-1.mdx:343]:
```suggestion
In your app.module.ts, import `IconService` & `IconModule` from `carbon-components-angular` module. IconModule will enable you to use the `ibmIcon` directive. `IconService` will enable you to register icons from `@carbon/icons` to be used with the directive.

<!-- prettier-ignore-start -->
```javascript path=src/app/app.module.ts
import { IconModule. IconService } from "carbon-components-angular";
<!-- prettier-ignore-end -->

Next, inject the service in the module class constructor and register it. There are 2 methods for icon registration `.register()` which accepts only one icon and `.registerAll()` which accepts array of icons. As we are going to use more than one icon, we are going to use the later method.

<!-- prettier-ignore-start -->
```javascript path=src/app/app.module.ts
export class AppModule {
  constructor(protected iconService: IconService) {
    iconService.registerAll([
      Notification20,
      UserAvatar20,
      AppSwitcher20
    ]);
  }
}
<!-- prettier-ignore-end -->
```

Breaking up information here to make it easier for users to absorb the steps & follow. Reasoning for not importing icons in header.component is given above.

File: src/pages/developing/angular-tutorial/step-1.mdx [hunk 2]
```diff
@@ -301,34 +301,46 @@ package.
...
-
-Now let's import the icons from our `@carbon/icons` package. In `app.module.ts`,
-`app.component.spec.ts` and `header.component.spec.ts`, we need to import each
-individual icon we will use and register them with the `IconService`
+Now let's import the icons from our `@carbon/icons` package. To not make our app very big, you don't need to import
+all the icons just those you would like to use. To do so let's open the `header.component.ts` file and import `Notification20` icon like below.
 
 <!-- prettier-ignore-start -->
-```javascript path=src/app/app.module.ts,src/app/app.component.spec.ts,src/app/header/header.component.spec.ts
+```javascript path=src/app/header/header.component.ts
```
[the remark this answers is not in the document]
Comment (Akshat55) [reply src/pages/developing/angular-tutorial/step-1.mdx:308]: Additionally, if you need to use the icons in multiple components in the same module, you will need to import & register icons again. This won't be necessary if imported & registered in module class.

**Review by Akshat55 — CHANGES_REQUESTED** [review]
Hi @plachy-jozef, thanks for contributing!

I have provided suggestions for steps which could be improved. Let me know if you have any questions or concerns regarding the suggestions.

Also, please run `yarn run format` to ensure formatting of the angular-tutorial `step-1.mdx` file is correct.

File: src/pages/developing/angular-tutorial/step-1.mdx (hunk 1 above)
Comment (plachy-jozef) [reply src/pages/developing/angular-tutorial/step-1.mdx]: thx for suggestions. I accepted them but not able to commit last suggestion. it's too long.
I didn't clone the repo on my local and created the PR just from web editor.

Comment (plachy-jozef) [reply src/pages/developing/angular-tutorial/step-1.mdx]: but obviously I will need to clone it as not able to run auto format. Will look into it today or following days. Thx for now.

Comment (Akshat55) [reply src/pages/developing/angular-tutorial/step-1.mdx]: Great, thanks! FYI, you'll need node v16.

**Review by dakahn — APPROVED** [review]

**Review by tay1orjones — APPROVED** [review]

**Pull request merged**